go.mod

```diff
@@ -78,6 +79,7 @@ require (
 	github.com/onsi/gomega v1.10.0 // indirect
 	github.com/openshift/api v0.0.0-00010101000000-000000000000
 	github.com/operator-framework/operator-sdk v0.0.0-00010101000000-000000000000
+	github.com/pborman/uuid v1.2.0 // indirect
```

Comment:
Do we need CQ?

Reply:
I am not sure. I didn't add this dependency, go.mod was updated automatically somehow.

Reply:
VSCode extension could update modify go.mod on the fly...

Reply:
if you depend on something new (and therefore so will CRW) then yes, please file a CQ.

Reply:
I've restored go.mod to prev state.